lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -30,142 +32,127 @@ type PropsWithoutAuth = {
 
 type PropsWithAuth = {
     requireAuth: true;
-    redirectToLogin: () => void;
+    redirectToLogin?: () => void;
 };
 
 type Props = (PropsWithoutAuth | PropsWithAuth) & {
     onSessionExpired?: () => void;
 };
 
+const noop = () => {
```

Comment:
We don't need this.. since it `requireAuth` is `true`, the user needs to provide `redirectToLogin`